packages/kokoas-e2e/cypress/e2e/invoice/issueInvoiceAndDiscard.cy.ts

```diff
@@ -0,0 +1,159 @@
+import { beforeEach, cy, describe, expect } from 'local-cypress';
+
+
+describe('請求書を発行する', () => {
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  const testCustGroupId = 'fe8029b9-4206-4344-a9d4-6d31918e8bb8';
+  const testReissueInvoiceId = '5a7a506f-e8b8-42f0-9437-d54c5d790701';
+
+  it('テスト準備[発行, 破棄]:APIから直接レコードを編集する', () => {
+    cy.task('prepareInvoice', 'KKB-C220020-01');
+  });
+
+  it('テスト準備[再発行]:APIから直接レコードを編集する', () => {
+    cy.task('prepareInvoiceReissue', testReissueInvoiceId);
+  });
+
+
+  it('請求書の動作確認：作成・更新・破棄', () => {
+    cy.visit(`/project/payment/invoice?custGroupId=${testCustGroupId}`);
+
+    // 使用する請求書にチェックを入れる
+    cy.get('input[name="estimates[1].isForPayment"]')
+      .as('isForPaymentInput')
+      .click();
+
+    // 請求金額を入力する
+    cy.get('input[name="estimates[1].billingAmount"]')
+      .as('billingAmountInput')
+      .type('80000', { delay: 100 });
+
+    // 入金予定日を設定する
+    // 未定のチェック
+    cy.get('input[name*="undecidedPaymentDate"]')
+      .as('undecidedPaymentDateInput')
+      .first()
```

Comment:
```suggestion
      .first()
      .as('undecidedPaymentDateInput')
```

これで、他のところに、first()を指定しなくてもいけると思います。

Reply:
ご指摘ありがとうございます。
[a1793f3](https://github.com/Lorenzras/yumecoco-monorepo/pull/173/commits/a1793f37f26a15c1b6681586927d3ae9580db1b1)にて修正しました。